src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -38,16 +41,13 @@ public void Dispose()
 
         public async Task<(Error, GitHubUser)> GetUserByLogin(string login)
         {
-            if (_rateLimitError != null)
-            {
-                return (_rateLimitError, null);
-            }
-
-            if (_unauthorizedError != null)
+            if (_fatalError != null)
             {
                 return default;
             }
 
+            Log.Write($"Calling GitHub user API to resolve {login}");
```

Comment:
Do we trace too much? We already trace them in GitHubUserCache?

Reply:
I moved them from GitHubUserCache to here because the log at GitHubUserCache does not always end up in a call to GitHub due to the return in line 46. I want this log to reflect the actual calls we made to GitHub.